### DISCUSSION TITLE
School wifi password

### DISCUSSION THREAD

**Discussion by DWPBEG** [3kh0/ext-remover#837] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
how do i get a wifi password on chromebook with chrome://network/

**Comment by 3kh0** [comment, marked as answer]
Use the password extractor tool, idk what the link is tho

**Reply by DWPBEG** [reply]
is it on github

**Reply by AshtonDavies** [reply]
https://sipe.glitch.me
https://luphoria.com/netlog-policy-password-tool

**Reply by DWPBEG** [reply]
they are both blocked

**Reply by AshtonDavies** [reply]
Figure something out and use your brain, because I guarantee that if it's an extension there is a bypass for it.

**Reply by DWPBEG** [reply]
yea imma see if i can disable the extension

**Reply by 3kh0** [reply]
https://cdn.3kh0.net/sipe

**Reply by DWPBEG** [reply]
i did it but they blocked the skiovox exploit lmao

**Reply by DWPBEG** [reply]
https://luphoria.com/netlog-policy-password-tool could you make this like what you did with the sipe one
